lib/src/components/NxSubmitMask/__tests__/NxSubmitMask.test.tsx

```diff
@@ -35,24 +34,23 @@ describe('NxSubmitMask', function() {
   });
 
   it('passes the message prop as the child of NxLoadingSpinner', function() {
-    expect(shallow(<NxSubmitMask/>).find(NxLoadingSpinner).children()).not.toExist();
-    expect(shallow(<NxSubmitMask message={null} />).find(NxLoadingSpinner).children()).not.toExist();
     expect(shallow(<NxSubmitMask message="foo" />).find(NxLoadingSpinner).children()).toHaveText('foo');
   });
 
-  it('passes the successMessage prop as the child of NxLoadingSpinner when the success prop is true', function() {
+  it('passes the successMessage prop as the success message text when the success prop is true', function() {
     const spinnerChildren = shallow(<NxSubmitMask message="foo" successMessage="bar" success />)
-        .find(NxLoadingSpinner).children();
+        .find('.nx-submit-mask__message .nx-submit-mask__message-text').children();
 
     expect(spinnerChildren).toHaveText('bar');
     expect(spinnerChildren).not.toHaveText('foo');
   });
 
```

Comment:
The new default text for the `NxLoadingSpinner` ("Submitting") needs a test

Reply:
> The new default text for the `NxLoadingSpinner` ("Submitting") needs a test

https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/b118888cce5bb67e6a8ce8fe9b33ed3f72b9c55f